app/javascript/src/Navigation/components/header/AccountSettingsMenu.jsx

```diff
@@ -1,6 +1,9 @@
-import React from 'react'
+// @flow
 
-const AccountSettingsMenu = ({accountSettingsLink, accountSettingsClass}) => (
+import * as React from 'react'
+import { ToolbarProps } from 'Navigation/components/header/types'
+
+const AccountSettingsMenu = ({accountSettingsLink, accountSettingsClass}: ToolbarProps): React.Node => (
```

Comment:
On the other hand, `ToolbarProps` looks wrong here. Since the component is `AccountSettingsMenu`, its props type should be `AccountSettingsMenuProps`. Does that make sense? We would have a conflict with `Toolbar`, since it has as props `{ toolbarProps, docsProps }` so I think `toolbarProps` are kinda wrong there and should be renamed.